tensor_test.go

```diff
@@ -123,3 +125,21 @@ func TestTensorPinMemory(t *testing.T) {
 		assert.Equal(t, " 1  2\n 3  4\n[ CPUFloatType{2,2} ]", b.String())
 	}
 }
+
+func TestTensorGC(t *testing.T) {
+	torch.GC()
+	runtime.LockOSThread()
+	c := make(chan bool, 0)
+	{
+		torch.NewTensor([][]float32{{1, 2}, {3, 4}})
+		go func() {
+			a := torch.NewTensor([][]float32{{1, 2}, {3, 4}})
+			c <- true
+			time.Sleep(time.Second)
+			runtime.KeepAlive(&a)
+		}()
+	}
+	<-c
+	assert.Eventually(t, func() bool { torch.GC(); return true }, 2*time.Millisecond, 10*time.Microsecond)
```

Comment:
Why do we need to call Eventually here? With the `<- c`, it seems that we can directly call `torch.GC()` here.

Reply:
We call `Eventually` here to make sure that `torch.GC()` doesn't have to wait the `Tensor a` in the goroutine to be unreachable.